lib/ansible/plugins/loader.py

```diff
@@ -49,7 +49,11 @@
     imp = None
 except ImportError:
     import imp
-    ModuleNotFoundError = None
+
+if not hasattr(__builtins__, 'ModuleNotFoundError'):
+    # this was introduced in Python 3.6
+    class ModuleNotFoundError(BaseException):
```

Comment:
should inherit from ImportError to keep same functionality

Reply:
*technically* this is not used anywhere: it's only defined in the case where python does not have this Exception yet, and then nothing will raise it.
but I still updated the code to be closer to reality :)